examples/work.yaml

```diff
@@ -0,0 +1,156 @@
+    calendar: https://alice.example/calendar-3D/
+
+  # alice finds the mfg request while she is searching for matches to her general offer
+  # she commits to do 3D printing work for the mfg project on Oct 14 sometime between 8h and 18h
+  # alice's calendar then will show her as unavailable during that time
+
+  - '@id': alice:33e8933b-ff73-4a01-964a-ca7a98893082
+    '@type': Commitment
+    action: work
+    provider: https://alice.example/
+    receiver: https://manufacturing.example/
+    resourceClassifiedAs: https://www.wikidata.org/wiki/Q229367 # 3D printing (as a type of work or skill)
+    committedQuantity:
+      qudt:unit: unit:Hour
+      qudt:numericValue: 8
+    committedTime:
+      '@type': time:ProperInterval
+      time:intervalStarts:
+        '@type': time:Instant
+        time:inXSDDateTimeStamp: 2018-10-14T08:00:00-5:00
+      time:intervalEnds:
+        '@type': time:Instant
+        time:inXSDDateTimeStamp: 2018-10-14T18:00:00-5:00
+
+  - '@id': mfg:6078f30a-b15b-4b1c-8e77-797abae11377
+    '@type': Satisfaction
+    satisfies: mfg:8e5fe80d-a769-4bd5-89e5-2136d33eab9s # mfg's intent (a request)
+    satisfiedBy: alice:33e8933b-ff73-4a01-964a-ca7a98893082 # the commitment
+    satisfiedQuantity:
+      qudt:unit: unit:Hour
+      qudt:numericValue: 8
+
+  - '@id': mfg:583e83d9-a46d-44ff-bd71-88513a1d83c9
+    '@type': Satisfaction
+    satisfies: alice:9bd19194-a36d-4a1f-896b-8082887962ca # alice's intent (an offer)
+    satisfiedBy: alice:33e8933b-ff73-4a01-964a-ca7a98893082 # the commitment
+    satisfiedQuantity:
+      qudt:unit: unit:Hour
+      qudt:numericValue: 8
+
+  # alice does the work on Oct 14 from 8h to 16h, completing the sequence
+
+  - '@id': mfg:60f4204e-b8d2-4026-8577-102c3f82c0ad
+    '@type': EconomicEvent
+    action: work
+    provider: https://alice.example/
+    receiver: https://manufacturing.example/
+    resourceClassifiedAs: https://www.wikidata.org/wiki/Q229367 # 3D printing (as a type of work or skill)
+    observedQuantity:
+      qudt:unit: unit:Hour
+      qudt:numericValue: 8
+    observedTime:
+      '@type': time:ProperInterval
+      time:intervalStarts:
+        '@type': time:Instant
+        time:inXSDDateTimeStamp: 2018-10-14T08:00:00-5:00
+      time:intervalEnds:
+        '@type': time:Instant
+        time:inXSDDateTimeStamp: 2018-10-14T16:00:00-5:00
+
+  - '@id': mfg:e56fd654-7b94-4d96-8e60-de39e08329a6
+    '@type': Fulfillment
+    fulfills: alice:33e8933b-ff73-4a01-964a-ca7a98893082 # the commitment
+    fulfilledBy: mfg:60f4204e-b8d2-4026-8577-102c3f82c0ad # the economic event
+    fulfilledQuantity:
+      qudt:unit: unit:Hour
+      qudt:numericValue: 8
+
+  # later on fablab finds alice's calendar and commits to make use of her skills during an available slot
+  # alice's calendar then shows her as unavailable during that slot
+
+  - '@id': fablab:86ad6e22-7640-4b93-972a-db7177ab2615
```

Comment:
I think always agent referenced as *provider* has to make the Commitment.

Reply:
Well interesting.  What if it is basically a plan?  Not a commitment for someone to give something?

Reply:
one can use `vf:Intent` that have specified both `vf:provider` and `vf:receiver`

Reply:
In this case, it is not an intent, it is agreed between both agents.  I think for usage where the receiver signed up for a slot on a schedule, they are committing to making use of that offer.  Might be based on offer or request.

Reply:
> What if it is basically a plan? Not a commitment for someone to give something?

`vf:Intent`

> In this case, it is not an intent, it is agreed between both agents.

`vf:Commitment` created by agent referenced as `vf:provider`

More *Conversation for Action* details we may need to handle soon, for now let's try to choose if the have already agreed and make *provider* create the Commitment, or they haven't agreed yet and then either one can create Intent.

Reply:
If an Intent (offer) is made with a schedule posted online that shows openings, and someone picks out a time duration from the schedule - doesn't that create a Commitment made by the receiver?  In fact there isn't a CfA really, no conversation is needed.

I don't think all plans are Intents.  Most plans include only Commitments because you don't need to seek out a matching agent.

>one can use `vf:Intent` that have specified both `vf:provider` and `vf:receiver`

Then how is it an Intent?  And how do you know if it is an offer or request?

Reply:
> Then how is it an Intent? And how do you know if it is an offer or request?

I think we will need http://dublincore.org/documents/dcmi-terms/#terms-creator

> doesn't that create a Commitment made by the receiver? In fact there isn't a CfA really, no conversation is needed.

I'll double check but in CfA only *performer* can make commitment to *customer*, in VF we would have *provider* making commitment to *receiver*. It makes sense to me since *receiver* benefits by receiving something they value, only *provider* can commit to do so since in most cases they will need to or did some efforts.

What problem do you see to at this stage omit CfA details and just have result where Alice makes that commitment?

Reply:
>What problem do you see to at this stage omit CfA details and just have result where Alice makes that commitment?

I see no problem at all.